pkg/search/search.go

```diff
@@ -78,7 +75,17 @@ func (o *RepoOptions) Run(logger log.Logger, args []string) error {
 	}
 
 	SortScore(res)
-	data, err := o.applyConstraint(res)
+	// TODO(itxaka): What happens if we pass more than one annotation? Should we check for all?
```

Comment:
we should either resolve that TODO or remove it from the PR.

Reply:
still struggling with this, will probably test limiting to just one `-a` flag

Reply:
Can you add a card to the board for that?

Reply:
This is outdated, doesnt exists anymore. Not sure why github doesnt automatically resolve this....

Reply:
cool, will resolve then :)